requirements.txt.lock

```diff
@@ -0,0 +1,18 @@
+appdirs==1.4.3
```

Comment:
very good using a lock file and a source file. pip-tools is an alternative (using requirements.in as source and requirements.txt as infile), but this works just as well :-D